app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,23 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(
+        self,
+        nickname: str,
+        musical_instrument: str,
+        bow_level: int
+    ) -> None:

```

Comment:
Please, preserve the same indentation in all your classes in the future, there should be no difference in indentation across different classes in your project